ansible_events_ui/api/__init__.py

```diff
@@ -603,3 +649,14 @@ async def authenticated_route(
     user: models.User = Depends(current_active_user),
 ):
     return {"message": f"Hello {user.email}!"}
+
+
+@router.get("/api/ssh_public_key")
```

Comment:
Please use dashes instead of underscores in API routes.

```suggestion
@router.get("/api/ssh-public-key")
```